src/components/html-injection/htmlInjectionHandlers.ts

```diff
@@ -20,14 +13,11 @@ export class HTMLInjectionHandlers implements IWidgetHandler {
             displayName: widgetDisplayName,
             iconClass: widgetIconClass,
 
-            /**
-             * This method invoked when the widget gets added to the content.
-             */
             createModel: async () => { 
                 const model = new HTMLInjectionModel();
                 model.htmlCode = htmlCodeInitial;
-                model.htmlCodeSizeStyles = htmlCodeSizeStylesInitial;
                 model.inheritStyling = true;
+                StyleHelper.setPluginConfigForLocalStyles(model.styles, "size", htmlCodeSizeStylesInitial);
```

Comment:
@azaslonov I've noticed you did not init the model.styles with any initial value. I guess we should use the height 500px & width 100%?

Reply:
Yes, you're right.

Reply:
(feel free to resolve this one)